components/pd_client/src/client.rs

```diff
@@ -643,7 +646,7 @@ impl PdClient for RpcClient {
                 .expect("expect region heartbeat sender");
             let ret = sender
                 .unbounded_send(req)
-                .map_err(|e| Error::Other(Box::new(e)));
+                .map_err(|_e|Error::Grpc(grpcio::Error::RemoteStopped));
```

Comment:
why drop the origin errors? can we unwrap the `Other Error`?

Reply:
fix